### ISSUE TITLE
[Request] Quick Chat via Ctrl+Click

### ISSUE THREAD

**Issue by ElvenChaos** [duke605/DiscordCE#5]
Please add an option to have Ctrl + Click on a channel name or user name auto-input the full command + id to chat to that channel/user as an alternative to having to use shift+click to append an ID into a command. This would make chatting in multiple settings or chatting to a channel other than the channel you are "in" faster and more accessible.

Thanks!

**Comment by duke605** [comment]
Unfortunately there is no ctrl + click insertion types. Only shift + click. :(

**Comment by ElvenChaos** [comment]
Do you mean none presently (viable feature) or that Ctrl+Click can't be programmed to do that?
If Ctrl+Click isn't feasible, could Shift+Click inherit these features when there is no other text in the user's chat input box, and retain the old features if there is already text in the chat input box?

Thank you for your time, I appreciate your help! :D

**Comment by duke605** [comment]
Unfortunately no :( there is no way to add conditions as it's all "black boxed" by minecrafts internal code. And to rewrite all that to substitute a few things would be an absolute nightmare as a lot of the code it obfuscated and just plain weird how it functions

**Comment by ElvenChaos** [comment]
Alright, my last idea: could Ctrl be treated like a "toggle" to switch what shift+click does? scheme would look like this:
Shift+Click: Quick Chat (inserts command + ID to talk to channel or DM)
Ctrl+Shift+Click: Old Shift+Click functionality (inserts just the relevant ID in case you want to use it for other commands)

This way, you are still just using "shift+click" to enter info, but ctrl alters what info is being processed. I don't know if this is how this works or not :D

**Comment by duke605** [comment]
Yes and no. The way it works is when a message come in it is given a insertion in json. So I could change that insertion to be different depending on a configuration but as soon as it's submitted to the chat the insertion is no longer change able. So pressing ctrl to change the insertion would only change the text inserted for the new messages, not messages up on the screen.

I know... It's VERY annoying that they don't provide a way to pass it a lambda or something...

**Comment by ElvenChaos** [comment]
Alright, thanks for taking the time to hear the request out! It is much appreciated!
